beacon-chain/utils/shuffle.go

```diff
@@ -10,30 +10,51 @@ import (
 )
 
 // ShuffleIndices returns a list of pseudorandomly sampled
-// indices. This is used to use to select attesters and proposers.
-func ShuffleIndices(seed common.Hash, validatorList []uint32) ([]uint32, error) {
-	// Since we are consuming 3 bytes of entropy at a time in the loop,
+// indices. This is used to shuffle validators on ETH2.0 beacon chain.
+func ShuffleIndices(seed common.Hash, indicesList []uint32) ([]uint32, error) {
+	// Each entropy is consumed from the seed in randBytes chunks.
+	randBytes := params.BeaconConfig().RandBytes
+	upperBound := 1<<(randBytes*8) - 1
```

Comment:
Please add some constant for the max num log2 of validators (24 or `log2(2^24)` if necessary for clarity) then replace 8 with MAX_VALIDATORS_LOG2 / randBytes.